### DISCUSSION TITLE
app_commands: suppressing "the application did not respond"

### DISCUSSION THREAD

**Discussion by codeofdusk** [Rapptz/discord.py#8457] [Q&A]
Hello,
I'm migrating to Discord.py 2.0/app commands.
In the previous `ext.commands` API, it was possible to perform an action without sending a message to the user. However, with app commands, if no message is sent, a "the application did not respond" ephemeral message is sent.
It seems like `interaction.response.defer(thinking=False)` is what I want here, but per the documentation app commands don't support it.
How can I accomplish this in the new API?
Thanks in advance!

**Comment by Rapptz** [comment]
Discord requires all application commands to respond. Best you can do is respond with an ephemeral message (by passing `ephemeral=True` to `InteractionResponse.send_message`)
